charts/atlantis/values.yaml

```diff
@@ -2,34 +2,43 @@
 # Values to override for your instance.
 ## -------------------------- ##
 
-# Provide a name to substitute for the full names of resources
+# -- Provide a name to substitute for the full names of resources.
 fullnameOverride: ""
 
-# Provide a name to substitute for the name of the chart
+# -- Provide a name to substitute for the name of the chart.
 nameOverride: ""
 
-## An option to override the atlantis url,
-##   if not using an ingress, set it to the external IP.
-# atlantisUrl: http://10.0.0.0
+# -- An option to override the atlantis url,
+# if not using an ingress, set it to the external IP.
+# Check values.yaml for examples.
+atlantisUrl: ""
+# Example:  http://10.0.0.0
 
-# Replace this with your own repo allowlist:
-orgAllowlist: <replace-me>
-# logLevel: "debug"
+# -- Replace this with your own repo allowlist.
+orgAllowlist: "<replace-me>"
 
-# Deprecated in favor of orgAllowlist
-# orgWhitelist:  <replace-me>
+# -- Deprecated in favor of orgAllowlist.
+orgWhitelist: "<replace-me>"
```

Comment:
@GMartinez-Sisti `orgWhitelist` seems to be taking precedence now that it is uncommented, breaking configs with `orgAllowlist` set.

```diff
          - name: ATLANTIS_REPO_ALLOWLIST
-           value: foo.bar/some/repo
+           value: <replace-me>
```

Reply:
Thanks for the heads up @peikk0! Fixing on https://github.com/runatlantis/helm-charts/pull/373.